Xamarin.Essentials/Contacts/Contacts.android.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Provider;
+using Net = Android.Net;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static async Task<Contact> PlatformPickContactAsync()
+        {
+            using var intent = new Intent(Intent.ActionPick);
+            intent.SetType(ContactsContract.CommonDataKinds.Phone.ContentType);
+            var result = await IntermediateActivity.StartAsync(intent, 101).ConfigureAwait(false);
```

Comment:
This code needs to be moved to that central location... Not sure if that change was merged yet though.

Reply:
This wasn't / isn't merged yet but making the change now